scanners/boostsecurityio/trivy-sbom/module.yaml

```diff
@@ -4,16 +4,60 @@ id: boostsecurityio/trivy-sbom
 name: BoostSecurity Trivy (FS SBOM)
 namespace: boostsecurityio/trivy-sbom
 
-
 config:
   support_diff_scan: false
 
+setup:
+  - name: download trivy
+    environment:
+      VERSION: 0.42.1
+      LINUX_X86_64_SHA: 305a4c6dbde3abac8e103376021e82c464478f829442e39a0fd980a77946bb22
+      LINUX_ARM64_SHA: cad74fc7c0a6a96ad0721987acda2e529b0b77db754c67e65bec8667b50d9ca5
+      MACOS_X86_64_SHA: a13af5f4d04b10ba181b12cc86f8735e441ab8eca7b6d23692b5061ae767411e
+      MACOS_ARM64_SHA: cebe7abc5d427f7bdedb506c2ae9b33259812ec5a779901b34efbac41bba572a
+    run: |
+      BINARY_URL="https://github.com/aquasecurity/trivy/releases/download/v${VERSION}"
+      ARCH=$(uname -m)
+
+      case "$(uname -sm)" in
+        "Linux x86_64")
+          BINARY_URL="${BINARY_URL}/trivy_${VERSION}_Linux-64bit.tar.gz"
+          SHA="${LINUX_X86_64_SHA} trivy.tgz"
+          ;;
+        "Linux aarch64")
+          BINARY_URL="${BINARY_URL}/trivy_${VERSION}_Linux-ARM64.tar.gz"
+          SHA="${LINUX_ARM64_SHA} trivy.tgz"
+          ;;
+        "Darwin x86_64")
+          BINARY_URL="${BINARY_URL}/trivy_${VERSION}_macOS-64bit.tar.gz"
+          SHA="${MACOS_X86_64_SHA} trivy.tgz"
+          ;;
+        "Darwin arm64")
+          BINARY_URL="${BINARY_URL}/trivy_${VERSION}_macOS-ARM64.tar.gz"
+          SHA="${MACOS_ARM64_SHA} trivy.tgz"
+          ;;
+        *)
+          echo "Unsupported machine: ${OPTARG}"
+          exit 1
+          ;;
+      esac
+
+      curl -o trivy.tgz -fsSL "${BINARY_URL}"
+      echo "${SHA}" | sha256sum --check
+
+      tar --no-same-owner -zxf trivy.tgz trivy
+      rm trivy.tgz
+      chmod +x trivy
 
 steps:
-    - scan:
-        command:
-            docker:
-                image: aquasec/trivy:0.37.3@sha256:906c909a441949317c252039ce8543ef79bcb9a59e7dd9b94e619191b82fedd2
-                command: fs --format=cyclonedx --license-full --cache-dir=/tmp/trivy/ .
-                workdir: /app
-        format: cyclonedx
+  - scan:
+      command:
+        run: |
+            $SETUP_PATH/trivy fs --format=cyclonedx --license-full --cache-dir=/tmp/trivy/ .
+      format: cyclonedx
+      post-processor:
+        docker:
+          image: public.ecr.aws/boostsecurityio/boost-scanner-trivy-sbom:2077b09@sha256:9acea8e0566becbbf19a51b7de326cda23d124254091f38cf5e2132f298a1301
+          command: process
+          environment:
+            PYTHONIOENCODING: utf-8
```

Comment:
Are you sure this change is right?

Why is the post-processor using a 6 month old commit?

Reply:
@Becojo this is to normalize the behavior of `trivy-sbom` and `trivy-sbom-image` the converter is removing duplicates https://github.com/boostsecurityio/boostsec-scanner-trivy-sbom/pull/2

Reply:
i know, i don't get why not use main if we're adding a converter to this module anyway. 

in any case, can be fixed forward.